src/Storefront/Controller/CmsController.php

```diff
@@ -64,6 +67,32 @@ public function page(?string $id, Request $request, SalesChannelContext $salesCh
         return $response;
     }
 
+    /**
+     * Basically the same as 'frontend.cms.page', but as a full page, not just the CMS page layout.
+     * This is achieved by using the storefront/page/content/index.html.twig template instead,
+     * which extends the base layout and is then using the detail.html.twig template internally.
+     */
+    #[Route(path: '/page/cms/{id}', name: 'frontend.cms.page.full', defaults: ['id' => null, 'XmlHttpRequest' => true, '_httpCache' => true], methods: ['GET', 'POST'])]
+    public function pageFull(?string $id, Request $request, SalesChannelContext $salesChannelContext): Response
```

Comment:
why can't we use `\Shopware\Storefront\Controller\NavigationController::index` or `\Shopware\Storefront\Controller\LandingPageController::index` ?

Reply:
I see, thanks for the explanation :+1: